src/draw/shapes.typ

```diff
@@ -454,7 +461,10 @@
     style.start = none
     style.symbol = none
 
-    let drawables = drawable.path((path-util.line-segment(pts),))
+    let (from, to) = (..pts)
+    let dir = vector.sub(to, from)
+
+    let drawables = drawable.path((path-util.line-segment((vector.sub(from, dir), from)),))
```

Comment:
This looks very confusing, whats wrong with `line-segment(from, vector.neg(to))`?

Reply:
How should that work?
The last argument has to be `from`, because that is where the end mark should be drawn.

I've reworked it, but we have to mirror `to` at `from` to make it work correctly.